lib/vagrant-qemu/driver.rb

```diff
@@ -56,9 +56,10 @@ def start(options)
           # basic
           cmd += %W(-machine #{options[:machine]})
           cmd += %W(-cpu #{options[:cpu]})
-          cmd += %W(-smp #{options[:smp]})
+          cmd += %W(-smp cpus=#{options[:smp]},sockets=1,cores=#{options[:smp]},threads=1)
```

Comment:
This may fail user who already set a complex "smp" config in there Vagrantfile. I think it's better to move this config to the Readme file as a suggestion?

Reply:
right

Reply:
okay, let me check quickly if that works on my setup, if so, I will move this to the readme section as suggested

Reply:
thanks a log